### ISSUE TITLE
[FEATURE] Sensible things

### ISSUE THREAD

**Issue by xXFreeFunXx** [JensvandeWiel/ArkAscendedServerManager#96]
I'm back from vacation and now it's time to rock ASASM :sunglasses: 

The following parameters are useless:
`?bRawSockets`
`?listen`
`-game`
`-log`
`-server`
`-nosteamclient`

I would provide these things via the INI and not via the start command:
```
?SessionName
?MultiHome
?QueryPort
?RCONEnabled
?RCONServerGameLogBuffer
?RCONPort
?MaxPlayers
?ServerAdminPassword
```

These only make sense in the start command if they cannot be manipulated by the users. Since we do not operate commercial servers, it would be tidier to obtain these parameters from the INI file.

The [ARK Wiki](https://ark.wiki.gg/wiki/Server_configuration) is very detailed and updates this page regularly.

**Comment by August2211** [comment]
Afaik QueryPort is useless in asa too

**Comment by xXFreeFunXx** [comment]
Yes, some things are not confirmed, but some things don't work.
These are listed here at [[SessionSettings]](https://ark.wiki.gg/wiki/Server_configuration#[SessionSettings])

**Comment by August2211** [comment]
> Yes, some things are not confirmed, but some things don't work. These are listed here at [[SessionSettings]](https://ark.wiki.gg/wiki/Server_configuration#%5BSessionSettings%5D)

QueryPort is for steam which ASA doesn't use

**Comment by xXFreeFunXx** [comment]
But there is -NitradoQueryPort but I don't know much about it.

**Comment by xXFreeFunXx** [comment]
`?Port` Must be in the start command, it is not enough if it is in the INI.
If it is only in the INI then the server starts with its own TCP port and UDP with the standard port 7777.

Therefore `?port` must also be in the start command so that the UDP port is taken over. Maybe another error, I don't know.

**Comment by JensvandeWiel** [comment]
closed by #103
